public_desktop/desktop-app.js

```diff
@@ -180,6 +180,16 @@ function startDesktopApp() {
 	}
 }
 
+// hard code wrongly interpreted words in spellchecker
+// electron has a bug with parsing of contractions
+// Ref: https://github.com/atom/electron/issues/1005
+function skipWords( locale, text ) {
+	// locale == 'en-us' already skipped in setup
+	var words = [ 'ain', 'couldn', 'didn', 'doesn', 'hadn', 'hasn', 'mightn', 'mustn', 'needn', 'oughtn', 'shan', 'shouldn', 'wasn', 'weren', 'wouldn'];
```

Comment:
Probably good to future proof this by anticipating other languages:

```
words = {
    "en-us": [ 'ain', ... ],
}

return ( words[ locale ] && text in words[ locale ] );
```

Because we'll inevitably forget about this function when we remove the `en-us` restriction :)

Also, bonus, move the array to a global so we're not generating the array every time `skipWords` is called.